### ISSUE TITLE
IS-ENES3 Analysis Platforms calls

### ISSUE THREAD

**Issue by jvegreg** [ESMValGroup/ESMValCore#967]
Just a simple question here: should we make a proposal to this call from IS-ENES?

https://portal.enes.org/data/data-metadata-service/analysis-platforms

I think it makes sense to try to get official access as ESMValTool developers to both platforms. Currently most of us have access to one of them and some of us even to both, but we leverage other projects for it. Having an official ESMValTool development project on both will make our situation a bit less precarious and will allow all of us to access both. This way dealing with platform related issues will be much easier as the most suitable person to fix them will always have access to the offending platform.

@ESMValGroup/technical-lead-development-team 
@ESMValGroup/scientific-lead-development-team

**Comment by valeriupredoi** [comment]
absolutely! Cheers for posting this @jvegasbsc :beer: This really looks down our alley and we should be able to apply - I can help with the application process on the CEDA-JASMIN side of things :+1: I think we need a new label called "fame and fortune awaits us" :grin:

**Comment by nielsdrost** [comment]
Hi @jvegasbsc and @valeriupredoi!

Sorry to rain on your parade but since we are part of IS-ENES we cannot actually submit to the IS-ENES call. I found out the hard way when I did exactly that last year.

The good news is that discussions around that submission led to the ESMValBot machine at DKRZ, so it worked out after all :-)

And I do agree to get access to both these machine for more developers, but this should be done by talking to representatives of the infrastructure directly, rather than going through this call.
